app/models/components/course/duplicator_component_host.rb

```diff
@@ -0,0 +1,15 @@
+class Duplicator::ModelComponentHost
```

Comment:
Nope, conceptually there's only one host if you mix behaviour in. If you have two hosts, you have... two hosts.